frontend/app-dashboard/js/view/panel-dashboard.js

```diff
@@ -176,7 +176,7 @@ define([
 
             let total_vulnerability_score = data['total_vulnerability_score'] ? data['total_vulnerability_score'].toFixed(2): 0;
             $('#vulnerability-score-' + element).html(total_vulnerability_score);
-            $('#'+ element +'-count').html(data['flooded_flooded_road_count']);
+            $('#'+ element +'-count').html(data['flooded_road_count']);
```

Comment:
Use standard column name conventions

Reply:
hi @lucernae that name is from the db? is db using that name now?

Reply:
Yes, I saw the frontend is using weird column name, so I check back if the backend actually have those name. It have those, so I fixed it in the backend too.